README.rst

```diff
@@ -0,0 +1,217 @@
+Pulls|
+
+The HistFactory p.d.f. template
+[`CERN-OPEN-2012-016 <https://cds.cern.ch/record/1456844>`__] is per-se
+independent of its implementation in ROOT and sometimes, it’s useful to
+be able to run statistical analysis outside of ROOT, RooFit, RooStats
+framework.
+
+This repo is a pure-python implementation of that statistical model for
+multi-bin histogram-based analysis and its interval estimation is based
+on the asymptotic formulas of “Asymptotic formulae for likelihood-based
+tests of new physics”
+[`arXiv:1007.1727 <https://arxiv.org/abs/1007.1727>`__]. The aim is also
+to support modern computational graph libraries such as PyTorch and
+TensorFlow in order to make use of features such as autodifferentiation
+and GPU acceleration.
+
+Hello World
+-----------
+
+.. code:: python
+
+   >>> import pyhf
+   >>> pdf = pyhf.simplemodels.hepdata_like(signal_data=[12.0, 11.0], bkg_data=[50.0, 52.0], bkg_uncerts=[3.0, 7.0])
+   >>> CLs_obs, CLs_exp = pyhf.infer.hypotest(1.0, [51, 48] + pdf.config.auxdata, pdf, return_expected=True)
+   >>> print('Observed: {}, Expected: {}'.format(CLs_obs, CLs_exp))
+   Observed: [0.05290116], Expected: [0.06445521]
+
+What does it support
+--------------------
+
+Implemented variations:
+  - [x] HistoSys
+  - [x] OverallSys
+  - [x] ShapeSys
+  - [x] NormFactor
+  - [x] Multiple Channels
+  - [x] Import from XML + ROOT via `uproot <https://github.com/scikit-hep/uproot>`__
+  - [x] ShapeFactor
+  - [x] StatError
+  - [x] Lumi Uncertainty
+
+Computational Backends:
+  - [x] NumPy
+  - [x] PyTorch
+  - [x] TensorFlow
+  - [x] JAX
+
+Available Optimizers
+
++---------------------+-----------------------+-----------------------+
+| NumPy               | Tensorflow            | PyTorch               |
++=====================+=======================+=======================+
+| SLSQP               | Newton’s Method       | Newton’s Method       |
+| (``scipy.optimize`` | (autodiff)            | (autodiff)            |
+| )                   |                       |                       |
++---------------------+-----------------------+-----------------------+
+| MINUIT              | .                     | .                     |
+| (``iminuit``)       |                       |                       |
++---------------------+-----------------------+-----------------------+
+
+Todo
+----
+
+-  ☐ StatConfig
+-  ☐ Non-asymptotic calculators
+
+results obtained from this package are validated against output computed
+from HistFactory workspaces
+
+A one bin example
+-----------------
+
+.. code:: python
+
+   nobs = 55, b = 50, db = 7, nom_sig = 10.
+
+.. image:: https://raw.githubusercontent.com/scikit-hep/pyhf/master/docs/_static/img/manual_1bin_55_50_7.png
+   :alt: manual
+   :width: 40 %
+
+.. image:: https://raw.githubusercontent.com/scikit-hep/pyhf/master/docs/_static/img/hfh_1bin_55_50_7.png
+   :alt: manual
+   :width: 40 %
+   :align: right
```

Comment:
```suggestion
.. image:: https://raw.githubusercontent.com/scikit-hep/pyhf/master/docs/_static/img/manual_1bin_55_50_7.png
   :alt: manual
   :width: 70 %
   :align: center

.. image:: https://raw.githubusercontent.com/scikit-hep/pyhf/master/docs/_static/img/hfh_1bin_55_50_7.png
   :alt: manual
   :width: 70 %
   :align: center
```

Reply:
I think that 70% across two lines is closer to what we currently have. The centering is a deviation from the current, but I like the look better.